libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -4,6 +4,7 @@ import { WrappedComponentProps, injectIntl, FormattedMessage } from "react-intl"
 import { CSSTransition, TransitionGroup } from "react-transition-group";
 import cx from "classnames";
 import { withTheme } from "@gooddata/sdk-ui-theme-provider";
+import differenceInCalendarDays from "date-fns/differenceInCalendarDays";
```

Comment:
Please swap lines 6 & 7 so that import statements of 3rd libraries come before import statements of GoodData's packages.